components/service/experiments/src/test/java/mozilla/components/service/experiments/ExperimentsTest.kt

```diff
@@ -73,7 +73,9 @@ class ExperimentsTest {
         // Initialize WorkManager (early) for instrumentation tests.
         WorkManagerTestInitHelper.initializeTestWorkManager(context)
 
-        configuration = Configuration()
+        // Setting the endpoint to a non-existent one to prevent actual experiments from being
+        // downloaded to tests.
+        configuration = Configuration(kintoEndpoint = "https://example.com")
```

Comment:
This domain actually exists and could lead to retrieving unexpected data or to unexpected delays.

Please change this to [`https://example.invalid`](https://serverfault.com/questions/846522/is-there-an-example-domain-which-is-guaranteed-not-to-exist)

Reply:
Changed to `.invalid`